docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -95,7 +124,23 @@ If you are using the old status check to block merging pull requests on GitHub,
 
 !!! info "This section applies to the repositories for which you set Codacy to post [coverage summaries](../../repositories-configure/integrations/github-integration.md#coverage-summaries) to your GitHub pull requests"
 
-On December 7th 2023 we set the new Coverage engine to post coverage summaries to GitHub, replacing the old Coverage engine. If you haven't done so yet, please [review and accept the updated Codacy app permissions](https://docs.github.com/en/enterprise-cloud@latest/apps/using-github-apps/reviewing-and-modifying-installed-github-apps#reviewing-permissions).
+{%
+    include-markdown "../../assets/includes/coverage-github-accept-permissions.md"
+    start="<!--accept-permission-start-->"
+    end="<!--accept-permission-end-->"
+%}
+
+On December 7th 2023 we set the new Coverage engine to post coverage summaries to GitHub, replacing the old Coverage engine.
+
+## Codacy app UI diff tabs from the new Coverage engine {: id="diff-tabs"}
+
+{%
+    include-markdown "../../assets/includes/coverage-github-accept-permissions.md"
+    start="<!--accept-permission-start-->"
+    end="<!--accept-permission-end-->"
+%}
+
+On January 1st 2024<!-- TODO ALA-789 date --> we set the [diff tab of the Coverage pull requests page](../../repositories-coverage/pull-requests.md#diff-tab) to get data from the new Coverage engine.
```

Comment:
Important: update with the actual launch date.

Reply:
Updated!